data/tokens.json

```diff
@@ -407,77 +454,77 @@
         "textCase": "{gtmds_text-case-none}",
         "textDecoration": "{gtmds_text-decoration-none}"
       },
       "type": "typography"
     },
     "paragraph_md": {
       "value": {
-        "fontFamily": "{gtmds_font-family-p}",
+        "fontFamily": "{gtmds_paragraph_font-family}",
         "fontWeight": "{gtmds_font-weight-400}",
-        "lineHeight": "{gtmds_line-height-base}",
-        "fontSize": "{Paragraph Size.gtmds_font-size-md}",
+        "lineHeight": "{gtmds_paragraph-md_line-height}",
+        "fontSize": "{Paragraph Size.gtmds_paragraph-md_font-size}",
         "letterSpacing": "{gtmds_letter-spacing-none}",
-        "paragraphSpacing": "{gtmds_paragraph-spacing-none}",
+        "paragraphSpacing": "{gtmds_paragraph-md_paragraph-spacing}",
         "textCase": "{gtmds_text-case-none}",
         "textDecoration": "{gtmds_text-decoration-none}"
       },
       "type": "typography"
     },
     "paragraph_md-accented": {
       "value": {
-        "fontFamily": "{gtmds_font-family-p}",
+        "fontFamily": "{gtmds_paragraph_font-family}",
         "fontWeight": "{gtmds_font-weight-500}",
-        "lineHeight": "{gtmds_line-height-base}",
-        "fontSize": "{Paragraph Size.gtmds_font-size-md}",
+        "lineHeight": "{gtmds_paragraph-md_line-height}",
+        "fontSize": "{Paragraph Size.gtmds_paragraph-md_font-size}",
         "letterSpacing": "{gtmds_letter-spacing-none}",
-        "paragraphSpacing": "{gtmds_paragraph-spacing-none}",
+        "paragraphSpacing": "{gtmds_paragraph-md_paragraph-spacing}",
         "textCase": "{gtmds_text-case-none}",
         "textDecoration": "{gtmds_text-decoration-none}"
       },
       "type": "typography"
     },
     "paragraph_sm": {
       "value": {
-        "fontFamily": "{gtmds_font-family-p}",
+        "fontFamily": "{gtmds_paragraph_font-family}",
         "fontWeight": "{gtmds_font-weight-400}",
-        "lineHeight": "{gtmds_line-height-base}",
-        "fontSize": "{Paragraph Size.gtmds_font-size-sm}",
+        "lineHeight": "{gtmds_paragraph-sm_line-height}",
+        "fontSize": "{Paragraph Size.gtmds_paragraph-sm_font-size}",
         "letterSpacing": "{gtmds_letter-spacing-none}",
-        "paragraphSpacing": "{gtmds_paragraph-spacing-none}",
+        "paragraphSpacing": "{gtmds_paragraph-sm_paragraph-spacing}",
         "textCase": "{gtmds_text-case-none}",
         "textDecoration": "{gtmds_text-decoration-none}"
       },
       "type": "typography"
     },
     "paragraph_sm-accented": {
       "value": {
-        "fontFamily": "{gtmds_font-family-p}",
+        "fontFamily": "{gtmds_paragraph_font-family}",
         "fontWeight": "{gtmds_font-weight-500}",
-        "lineHeight": "{gtmds_line-height-base}",
-        "fontSize": "{Paragraph Size.gtmds_font-size-sm}",
+        "lineHeight": "{gtmds_paragraph-sm_line-height}",
+        "fontSize": "{Paragraph Size.gtmds_paragraph-sm_font-size}",
```

Comment:
@StefanIvanov Is the space in the category alright? "Paragraph Size" instead of "Paragraph-size" or something like that?

Reply:
It is defined as 

"Paragraph Size": {
      "gtmds_paragraph-lg_font-size": {
        "value": "{gtmds_font-size-base}+{gtmds_size-xs}",
        "type": "fontSizes"
      },
      "gtmds_paragraph-md_font-size": {
        "value": "{gtmds_font-size-base}",
        "type": "fontSizes"
      },
      "gtmds_paragraph-sm_font-size": {
        "value": "{gtmds_font-size-base}-{gtmds_size-xs}",
        "type": "fontSizes"
      }
    },